### PR TITLE
OSDOCS-1209: Adding OpenShift control plane services including the OA…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…uth API server

https://issues.redhat.com/browse/OSDOCS-1209

Preview: https://osdocs-1209--ocpdocs.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#defining-masters_control-plane

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@rhpmali Can you please review these descriptions?

Preview: https://osdocs-1209--ocpdocs.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#defining-masters_control-plane

File: modules/architecture-machine-roles.adoc [hunk 1]
```diff
@@ -58,28 +58,51 @@ master static Pods and etcd static Pods working on the same hosts.
...
 .Kubernetes services that run on the control plane
-
-[options="header"]
+[cols="1,2",options="header"]
 |===
 |Component |Description
-|API Server
+|Kubernetes API server
 |The Kubernetes API server validates and configures the data for Pods, Services,
 and replication controllers. It also provides a focal point for cluster’s shared
```
Comment (adellape) [inline modules/architecture-machine-roles.adoc:69]: An old sentence, but I guess this was supposed to be saying "for a cluster's shared state"? Though may be better to rewrite as "the shared state of a cluster", per IBM SG's guidance on avoiding inanimate possessives.

File: modules/architecture-machine-roles.adoc [hunk 2]
```diff
@@ -58,28 +58,51 @@ master static Pods and etcd static Pods working on the same hosts.
...
 state.
 |etcd
 |etcd stores the persistent master state while other components watch etcd for
 changes to bring themselves into the specified state.
 //etcd can be optionally configured for high availability, typically deployed with 2n+1 peer services.
-|Controller Manager Server
-|The Controller Manager Server watches etcd for changes to objects such as
+|Kubernetes controller manager
+|The Kubernetes controller manager watches etcd for changes to objects such as
 replication, namespace, and serviceaccount controller objects, and then uses the
```
Comment (adellape) [inline modules/architecture-machine-roles.adoc:77]: s/serviceaccount/service account/

?

**Review by adellape — APPROVED** [review]
:fire:

**Pull request merged**